packages/orbit-components/src/TileGroup/TileGroup.stories.tsx

```diff
@@ -14,272 +15,265 @@ import TileGroup from ".";
 
 const getIcon = source => Icons[source];
 
-export default {
+type TileGroupPropsAndCustomArgs = React.ComponentProps<typeof TileGroup> &
+  React.ComponentProps<typeof Tile> & {
+    content: string;
+  };
+
+const meta: Meta<TileGroupPropsAndCustomArgs> = {
   title: "TileGroup",
+  component: TileGroup,
 };
 
-export const DefaultJustWrapper = ({ dataTest, as, content }) => {
-  return (
-    <TileGroup as={as} dataTest={dataTest}>
+export default meta;
+type Story = StoryObj<TileGroupPropsAndCustomArgs>;
+
+export const DefaultJustWrapper: Story = {
+  render: ({ content }) => (
+    <TileGroup>
       <Tile onClick={action("clicked")}>{content}</Tile>
       <Tile onClick={action("clicked")}>{content}</Tile>
       <Tile onClick={action("clicked")}>{content}</Tile>
       <Tile onClick={action("clicked")}>{content}</Tile>
     </TileGroup>
-  );
-};
-
-DefaultJustWrapper.story = {
-  name: "Default - just wrapper",
+  ),
 
   parameters: {
-    info: "This is the default configuration of this component.",
+    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-DefaultJustWrapper.args = {
-  dataTest: "test",
-  as: "div",
-  content: "Lorem ipsum dolor sit amet",
+  args: {
+    content: "Lorem ipsum dolor sit amet",
+  },
 };
 
-export const DefaultWithHeaderProps = ({ title, description, icon }) => {
-  const Icon = getIcon(icon);
-  return (
-    <TileGroup>
-      <Tile
-        onClick={action("clicked")}
-        icon={Icon && <Icon />}
-        title={title}
-        description={description}
-      />
-      <Tile
-        onClick={action("clicked")}
-        icon={Icon && <Icon />}
-        title={title}
-        description={description}
-      />
-      <Tile
-        onClick={action("clicked")}
-        icon={Icon && <Icon />}
-        title={title}
-        description={description}
-      />
-      <Tile
-        onClick={action("clicked")}
-        icon={Icon && <Icon />}
-        title={title}
-        description={description}
-      />
-    </TileGroup>
-  );
-};
+export const DefaultWithHeaderProps: Story = {
```

Comment:
I would add the `noHeaderIcon` arg as it's related to the "header props".

Reply:
I didn't want to add any more of `Tile` args as, in the end, it's `TileGroup` story. But as agreed, I kept only the Playground and RTL story and since `header` is hardcoded there as ReactNode, I kept only `children` arg from `Tile` component. I hope it makes sense like this.